app/src/main/java/com/example/exercise3/AlarmReceiver.java

```diff
@@ -0,0 +1,26 @@
+package com.example.exercise3;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+public class AlarmReceiver extends BroadcastReceiver {
+
+    QuotesGenerator qg = new QuotesGenerator();
```

Comment:
you could have made getQuote static and then you didn't need to create an instance of QuotesGenerator to use it.
not a big deal but you can think on it for next time